dub.selections.json

```diff
@@ -21,4 +22,4 @@
 		"vibe-core": {"path":"submodules/vibe-core/"},
 		"vibe-d": {"path":"submodules/vibe.d/"}
 	}
-}
+}
```

Comment:
Why did this happen again? Didn't you fix your text editor?

Reply:
There was a big change in the upstream, It fell out while working rebase. ;)